Makefile

```
@@ -288,3 +287,22 @@ build-reproducible:
 		--name latest-build cosmossdk/rbuilder:latest
 	docker cp -a latest-build:/home/builder/artifacts/ $(CURDIR)/
 .PHONY: build-reproducible
+
+# Implements test splitting and running. This is pulled directly from
+# the github action workflows for better local reproducibility.
+
+GO_TEST_FILES := $(shell find $(CURDIR) -name "*_test.go")
```

Could also be:
```suggestion
GO_TEST_FILES != find $(CURDIR) -name "*_test.go"
```
YMMV